### ISSUE TITLE
Add a warning for download the base nextflow binary for clusters

### ISSUE THREAD

**Issue by abhi18av** [seqeralabs/nf-tower-docs#89]
QQ: Should this note be added for each of the clusters individually? Or only the SLURM one?

**Comment by pditommaso** [comment]
All CE that rely on user-provided nextflow ie. Slurm, GridEngine, etc

**Comment by abhi18av** [comment]
Okay, I'm planning to add the following warning at the top. Let me konw if you'd like to tweak


> Please use the self-installer binary version of Nextflow, instead of the pre-packaged binary. For example, use [nextflow](https://github.com/nextflow-io/nextflow/releases/download/v21.04.2/nextflow) instead of [nextflow-VERSION-all](https://github.com/nextflow-io/nextflow/releases/download/v21.04.2/nextflow-21.04.2-all).

**Comment by pditommaso** [comment]
Think it's related to this issue https://github.com/nextflow-io/nextflow/issues/2145

**Comment by abhi18av** [comment]
> Nailed! Thanks a lot for reporting this issue 👍 
> https://github.com/nextflow-io/nextflow/issues/2145#issuecomment-882676557

Ah, I guess this mean that we don't need to mention in the docs at th moment 🤔 

Perhaps from future perspective, when we want to allow users to use any NF version - would the `nf-edge-all` release be able to adapt the listed plugin versions?

**Comment by pditommaso** [comment]
Yeah, exactly. don't think it's a frequent problem, let's keep an eye on it with future versions it should not happen anymore.
